### DISCUSSION TITLE
Using a fake data to create lag

### DISCUSSION THREAD

**Discussion by guguxh** [wxyz-abcd/node-haxball#53] [General]
This is possible? Like to simulate a red bar, max ping and these things.

**Comment by wxyz-abcd** [comment]
Yes, it should be possible. The host sends/receives ping messages from/to clients. If this message is sent with random delays, or sometimes not sent at all, it might possibly cause this behaviour, but the gameplay will also possibly start to suffer because basro is using those ping values to calculate how many frames the physics engine has to run on each render cycle. I will try this sometime later.
